### ISSUE TITLE
Why didn't you fork instead of nuking all history?

### ISSUE THREAD

**Issue by fr500** [masterfeizz/DeSmuME-NX#2]
^^ 

This makes it impossible to upstream, or to improve it..

**Comment by JoshuaDoes** [comment]
I agree, the full commit history should be there from a fork (or even a new GitHub repo but using the original repo, Google is your friend on changing push URLs) so the process of porting things is shown and also so people can revert changes or upstream or anything else
